rootex/framework/components/visual/camera_component.cpp

```diff
@@ -171,7 +171,14 @@ JSON::json CameraComponent::getJSON() const
 void CameraComponent::addCustomPostProcessingDetails(const String& path)
 {
 	PostProcessSystem::GetSingleton()->addCustomPostProcessing(path);
-	m_PostProcessingDetails.customPostProcessing.insert({ path, true });
+	if (m_PostProcessingDetails.customPostProcessing.find(path) == m_PostProcessingDetails.customPostProcessing.end())
+	{
```

Comment:
This if statement is not needed. You are doing the same operation both ways. [path] will insert if the key is not already there

Reply:
Fixed